packages/utils/test/api-client.test.js

```diff
@@ -51,7 +51,9 @@ describe('Lighthouse CI API Client', () => {
       await client.getBuilds('124', {branch: 'master'});
       expect(fetchMock).toHaveBeenCalledWith(
         `http://localhost:9000/v1/projects/124/builds?branch=master`,
-        {headers: {}}
+        {
```

Comment:
revert this one?

Reply:
This happen with linter (—fix) but es I can revert that